scripts/test/playback_gif.py

```diff
@@ -0,0 +1,75 @@
+import os
+from pathlib import Path
+import argparse
+import numpy as np
+import cv2
+from PIL import Image
+import imageio
+
+
+parser = argparse.ArgumentParser(description='format')
+parser.add_argument('--dir', type=str, default= 'vid2vid/datasets/sceneNet', help='Dataset directory')
+parser.add_argument('--res', type=str, default= 'vid2vid/results/depth2room_320_8g', help='Result directory')
+parser.add_argument('--num', type=int, default= 1, help='Number of result versions')
+
+
+def depth_norm(img, depth_min = 0.0, depth_max = 5.0):
+
+    img = 255.0*(1.0 - (img - 1000.0*depth_min) / (1000.0*(depth_max - depth_min)))
+    return img.astype(np.uint8)
+
+ 
+def load_img(test_A, test_B, result, indx, seq_len):
```

Comment:
what does result contain? looks like an dir of some sort, add doc-string please? or rename with dir in variable name
indx looks like num of channels to differentiate gray vs RGB?